src/main/java/org/jboss/pnc/bifrost/kafkaconsumer/MessageConsumer.java

```diff
@@ -84,7 +86,16 @@ void initMetrics() {
     @Incoming("logs")
     @Transactional
     @WithSpan()
-    public void consume(@SpanAttribute(value = "json") String json) {
+    public void consume(@SpanAttribute(value = "json") List<String> jsonBatchValues) {
```

Comment:
Can we somehow set it up so that we do not have like a million at once if the queue is congested? Like a 5000 limit. Also, I once dabbled with batch inserts and by default they are not batched [1]. Setting `hibernate.jdbc.batch_size` or Panache equivalent would make this go the speed of light.

[1] https://www.baeldung.com/jpa-hibernate-batch-insert-update#default-behaviour

Reply:
Hell Yeah! That's a great suggestion!

Reply:
PR: https://github.com/project-ncl/bifrost/pull/409